apps/console/src/index.tsx

```diff
@@ -79,23 +80,25 @@ const RootWithConfig = (): ReactElement => {
     }
 
     return (
-        <ThemeProvider theme={ AsgardeoTheme } defaultMode="light" modeStorageKey="console-oxygen-mode">
-            <Provider store={ store }>
-                <BrowserRouter>
-                    <AuthProvider
-                        config={ AuthenticateUtils.getInitializeConfig() }
-                        fallback={ <PreLoader /> }
-                        getAuthParams={ getAuthParams }
-                    >
-                        <AppConfigProvider>
-                            <OrganizationsProvider>
-                                <ProtectedApp />
-                            </OrganizationsProvider>
-                        </AppConfigProvider>
-                    </AuthProvider>
-                </BrowserRouter>
-            </Provider>
-        </ThemeProvider>
+        <AppSettingsProvider>
+            <ThemeProvider theme={ AsgardeoTheme } defaultMode="light" modeStorageKey="console-oxygen-mode">
```

Comment:
Shall we avoid using product names for variables?

Reply:
Let's track this in a different issue.